crates/maple_cli/src/cmd/cache.rs

```diff
@@ -72,13 +72,9 @@ impl CacheEntry {
     /// Construct the cache entry given command arguments and its working directory, the `total`
     /// info is cached in the file name.
     pub fn new(cmd_args: &[&str], cmd_dir: Option<PathBuf>, total: usize) -> Result<PathBuf> {
-        let mut dir = clap_cache_dir();
-        dir.push(cmd_args.join("_"));
-        if let Some(mut cmd_dir) = cmd_dir {
-            dir.push(format!("{}", calculate_hash(&mut cmd_dir)));
-        } else {
-            dir.push("no_cmd_dir");
-        }
+        let cmd_dir = cmd_dir.unwrap_or(PathBuf::from("no_cmd_dir"));
```

Comment:
Are you sure, that `no_cmd_dir` would not exist? Probably you should just go with

```rust
if let Some(cmd_dir) = cmd_dir {
    /* go code brr */
}
```

Reply:
Not sure what you mean here. Also I've removed all code dealing with caching for tagfiles as it's not useful in this case. Should I revert that?